ui/cypress/support/commands.js

```diff
@@ -80,3 +83,35 @@ Cypress.Commands.add('setSelection', { prevSubject: true }, (subject, query, end
     }
   })
 ));
+
+
+// let sharedStatus = 0;
+
+// Cypress.Commands.add('spyPoll', {}, (method, url, property, expectedVal, nth=0) => {
```

Comment:
Previous attemp to loop and wait until, for example, an elwood job completed.

Freakin nightmare. I opted to use a separate helpers.js function with async/await and do `cy.request()`s, then use `cy.wrap()` to wait for this promise in tests.

Help needed to consider replacing the `cy.request()` to `axios` and internal helpers.js `cy.wait()` to a `sleep()` function that uses promises, maybe.